c_plus_plus/src/view/views/recording_view.h

```diff
@@ -1,6 +1,11 @@
 #ifndef RECORDING_VIEW_H
 #define RECORDING_VIEW_H
 
+#include <QCamera>
+#include <QCameraInfo>
+#include <QCameraViewfinder>
+#include <QListWidgetItem>
+
```

Comment:
Use forward declaration to make header file lighter.
Call `class QCamera;` in the header file and `#include <QCamera>` in the implementation file.The same can be applied to all qt include.